bitnami/mysql/values.yaml

```diff
@@ -150,6 +150,10 @@ auth:
   ##   replicator: /vault/secrets/mysql-replicator
   ##
   customPasswordFiles: {}
+  ## @param auth.defaultAuthenticationPlugin Sets the default authentication plugin, by default it will use `mysql_native_password`
+  ## NOTE: `mysql_native_password` will be deprecated in future mysql version and it is used here for compatibility with previous version. If you want to use the new default authentication method set it to `caching_sha2_password`.
+  ##
+  defaultAuthenticationPlugin: {}
```

Comment:
Is there any reason to use an object as default value instead of a string? It seems the value should be a string.